apps/notification/receivers/project_membership.py

```diff
@@ -7,14 +7,13 @@
     ProjectMembership,
     ProjectRole,
 )
-from project.serializers import ProjectJoinRequestSerializer
+from project.serializers import ProjectJoinGqSerializer
 
 
 @receiver(post_save, sender=ProjectJoinRequest)
 def create_notification(sender, instance, created, **kwargs):
     admins = instance.project.get_admins()
-    data = ProjectJoinRequestSerializer(instance).data
-
+    data = ProjectJoinGqSerializer(instance).data
```

Comment:
Let's not change this.

For Graphql we only need writable fields. So remove read-only fields from `ProjectJoinGqSerializer` and use `ProjectJoinRequestSerializer` here.